refactoring/initial_two_classes.py

```diff
@@ -32,17 +32,27 @@ def add_person(self, name, age, job):
 
     def number_of_connections(self, name):
         """Find the number of connections that a person in the group has"""
-        pass
+        return len(self.connections[name])
```

Comment:
Could also have another guard clause here
```suggestion
        if not self.contains(name):
            raise ValueError(f"I don't know about {name}.")
        return len(self.connections[name])
```